### ISSUE TITLE
can not open Terminal In windows10

### ISSUE THREAD

**Issue by RachelChanning** [de-jcup/eclipse-bash-editor#178]
![OO 153DIFZ8PSN6(5$3RZM1](https://user-images.githubusercontent.com/36948289/68942764-50ef0100-07e4-11ea-9ff0-91c19aa2b65b.png)
 
hello,when I click the 'Test Terminal' button ,then the terminal window opened then quickly closed.
the version eclipse I used is 2019-09 R (4.13.0),and installed  Bash Editor version is 2.2.2. At the beginning of install it ,the eclipse show me this 
![image](https://user-images.githubusercontent.com/36948289/68943091-2cdfef80-07e5-11ea-8211-9b8da7903528.png)
and I select the "Install anyway" selection ,dont know is this has any influence.
I m so lucky found this plugin ,such a helpful plugin.very very thanks for reply me !thanks thanks.

**Comment by de-jcup** [comment]
Hello @RachelChanning , the warning about usigned content is "okay" because I currently do not sign my plugins (but maybe in future).

I tried out at a windows 10 machine - and you are right there are problems.

## Problem0:. Initial setup
As mentioned at 
https://github.com/de-jcup/eclipse-bash-editor/wiki/Debugger#section-windows you need to install a windows bash variant, without this, it can never work.

So on my Windows machine I had to download git for windows at  https://git-scm.com/download/win
and install it.

## Problem1 : default setup not correct 
First of a BashEditor 2.2.2 did not bring up correct default for start command, should not end with `CMD_TERMINAL` but `${BE_CMD_CALL}`

## Problem2: Path separator
After changing the start command in preferences to `cmd.exe /C start "CMD_TITLE" cmd.exe /C "${BE_CMD_CALL}"` enabling debugging to console and testing the window it was failing and the output in console looked like:
```
cmd.exe /C start "CMD_TITLE" cmd.exe /C "cd C:\Users\yourUser\AppData\Local\Temp;./terminallaunch4382865237318220812.sh -a 1 -b 2;_exit_status=$?;echo "Exit code=$_exit_status";read -p "Press enter to continue...""
```
The problem is that ";" is not (any longer ?) a correct  path separator!

Setting & instead ; separator does work:
```
cmd.exe /C start "CMD_TITLE" cmd.exe /C "cd C:\Users\yourUser\AppData\Local\Temp & terminallaunch4382865237318220812.sh -a 1 -b 2;_exit_status=$?;echo "Exit code=$_exit_status";read -p "Press enter to continue...""
```

**Comment by de-jcup** [comment]
@RachelChanning as long as  this is not fixed - I am extreme busy at the moment - I would suggest you try WSL (Windows Subsystem for Linux) this should work very well - see https://github.com/de-jcup/eclipse-bash-editor/issues/155 where a user sent me a mail.

Unfortunately I got no response inside the issue - so please tryout.

**Comment by de-jcup** [comment]
@RachelChanning I decided to put this on a hotfix release because its an major issue at windows.

**Comment by de-jcup** [comment]
@RachelChanning I just released 2.2.3 on eclipse marketplace and it works now for windows 10 out of the box.

Preferences have been changed  and look now different. Windows
![image](https://user-images.githubusercontent.com/11644753/69000648-2f207780-08d3-11ea-9ff7-801f3db6b313.png)

**Comment by de-jcup** [comment]
@RachelChanning I just released 2.2.3 on eclipse marketplace and it works now for windows 10 out of the box.

Preferences have been changed  and look now different for windows - does now work with Windows 10 out of the box when you have installed a bash variant availabe from PATH environment variable. 
(see also https://github.com/de-jcup/eclipse-bash-editor/wiki/Debugger#section-windows)
![image](https://user-images.githubusercontent.com/11644753/69001109-a1945600-08d9-11ea-9a46-a57cfb90d361.png)

**Comment by de-jcup** [comment]
Added a video at https://youtu.be/Qwjx5XFFRFw showing bash debugger v2.2.3 in action on a Windows 10 machine.
